PIL/Image.py

```diff
@@ -1262,6 +1335,7 @@ def getprojection(self):
         return [i8(c) for c in x], [i8(c) for c in y]
 
     def histogram(self, mask=None, extrema=None):
+        # type: (Optional[Image], Optional[Any]) -> List[int]
```

Comment:
Instead of `Optional[Any]`, which resolves to the same thing as `Any` I believe, perhaps this should be `Optional[ImagingCore]` ? Also, the docstring doesn't have a corresponding `:param` mentioning this parameter.